src/main/java/bio/terra/cli/service/utils/HttpUtils.java

```diff
@@ -294,12 +296,13 @@ public static HttpResponse sendHttpRequest(
       logger.info("Caught possible one-time error: {}", ex);
 
       // handle the one-time error
-      handleOneTimeError.makeRequest();
+      callWithRetries(handleOneTimeError, isRetryable);
```

Comment:
I don't think we should conflate what's retryable for handleOneTimeError and makeRequest. Arguably, this function shouldn't do retries at all and the retries should be handled in the `SupplierWithCheckedException` if necessary.

Reply:
Hmm I'm conflicted on this one. On the one hand, I agree the `handleOneTimeError` pattern is distinct from retries, but everywhere we're using this pattern in the codebase we also need retries. Polling was a similar case, hence the `pollWithRetries` function. So I think it makes sense to keep the retries together with the pattern, until we have a case that needs them separated.

Good point about separating `isRetryable` for the initial request and the one-time error handling functions. Will take a pass at that.